traffic_ops/testing/api/v3/profiles_test.go

```diff
@@ -30,18 +30,47 @@ func TestProfiles(t *testing.T) {
 	WithObjs(t, []TCObj{CDNs, Types, Profiles, Parameters}, func() {
 		CreateBadProfiles(t)
 		UpdateTestProfiles(t)
+		currentTime := time.Now().UTC().Add(-5 * time.Second)
+		time := currentTime.Format(time.RFC1123)
+		var header http.Header
+		header = make(map[string][]string)
+		header.Set(rfc.IfUnmodifiedSince, time)
+		UpdateTestProfilesWithHeaders(t, header)
 		GetTestProfilesIMS(t)
 		GetTestProfiles(t)
 		GetTestProfilesWithParameters(t)
 		ImportProfile(t)
 		CopyProfile(t)
+		header = make(map[string][]string)
+		etag := rfc.ETag(currentTime)
+		header.Set(rfc.IfMatch, etag)
+		UpdateTestProfilesWithHeaders(t, header)
 	})
 }
 
+func UpdateTestProfilesWithHeaders(t *testing.T, header http.Header) {
+	firstProfile := testData.Profiles[0]
+	// Retrieve the Profile by name so we can get the id for the Update
+	resp, _, err := TOSession.GetProfileByName(firstProfile.Name, header)
+	if err != nil {
+		t.Errorf("cannot GET Profile by name: %v - %v", firstProfile.Name, err)
+	}
+	remoteProfile := resp[0]
+	remoteProfile.Description = "UPDATED"
```

Comment:
To cover a corner-case where the description was already "UPDATED", this should append with `+= "UPDATED"` so that it's always changed.

Reply:
On the other hand, I don't know if that matters for the purposes of this test. But if it doesn't, then why bother changing it at all?